tests/run-macros/tasty-create-method-symbol/Macro_1.scala

```diff
@@ -27,7 +27,7 @@ object Macros {
 
     // test for no argument list (no Apply node)
     val sym2 : Symbol = Symbol.newMethod(
-      Symbol.currentOwner,
+      Symbol.spliceOwner,
```

Comment:
This usage will be error-prone in a nested context, becuase `Symbol` is always imported by `qctx.reflect._`, instead of the nested context.

Reply:
Indeed. I need to improve this interface. Note that the same happens with `currentOwner` in a more complex way as it looks for an implicit context that might come from the imported or implicitly available one, depending on how it is imported. This version is slightly more stable but still not perfect. I would fix this in a separate PR because it is an orthogonal concern.